pages/components/Footer/Footer.js

```diff
@@ -8,15 +8,11 @@ import { useState } from "react" // Импортируем useState
 const Footer = () => {
   const [ email, setEmail ] = useState("") // Создаем состояние для хранения значения поля email
   const [ emailStatus, setEmailStatus ] = useState(false)
+  const [ validation, setValidation ] = useState(false)
+
   const handleClick = (e) => {
     e.preventDefault()
-
-    if ( isValidEmail(email) ) {
-      setEmailStatus(true)
-      console.log("Email отправлен:", email)
-    } else {
-      alert("Пожалуйста, введите действительный E-mail.")
-    }
+    isValidEmail(email) ? setEmailStatus(true) : setValidation(true)
```

Comment:
I totally do not understand what and why happens when you're validation email.

`isValidEmail(email) ? submitEmail() : showEmailValidationError()`

Reply:
Thanks, fixed